tests/testthat/test-logistf.R

```diff
@@ -1,6 +1,7 @@
 if (requiet("testthat") &&
```

Comment:
```suggestion
# nocov start

if (requiet("testthat") &&
```

Reply:
aren't these comments used in the R files instead of the tests? do you have an example of this in other repos?

Reply:
https://github.com/r-lib/covr#exclusion-comments

Reply:
`testthat::skip_on_covr()` would work to skip the tests but what we need is to prevent `logistf` from being loaded when covr runs. This is why I initially put `!covr::in_covr`, the goal was to skip the whole file

Reply:
actually, putting `testthat::skip_on_covr()` at the top of the file seems to work